### PR TITLE
Fix Plot dataframe and plot profit

### PR DESCRIPTION
**Pull request by Alocaly**
Thank you for sending your pull request. But first, have you included
unit tests, and is your code PEP8 conformant? [More details](https://github.com/gcarq/freqtrade/blob/develop/CONTRIBUTING.md)

## Summary
Fix Plot dataframe and plot profit

Solve the issue: #___

## Quick changelog

I changed in backtesting row.date.strftime('%s'), that is not standard - depends on the OS region -, and does not work for me, to generate correct backtest result.
And fixes some merge issues with Plot Dataframe and plot Profit...

### CODE REVIEW

**Comment by gcarq** [comment]
Thanks a lot! Please check your code with flake8, there are some formatting issues otherwise LGTM

**Comment by Alocaly** [comment]
Sorry, actually, I need to learn how to use github, the second commit ( the one non compliant ) was intended for my repo only.
It's far from being something to release for now...
I don't know why it ended here :(

File: scripts/plot_dataframe.py [hunk 1]
```diff
@@ -87,7 +83,7 @@ def plot_analyzed_dataframe(args: Namespace) -> None:
     )
 
     df_buy = data[data['buy'] == 1]
-    buys = go.Scattergl(
+    buys = go.Scatter(
```
Comment (vertti) [inline scripts/plot_dataframe.py]: Why this change? It makes the whole graph drastically slower by removing the OpenGL acceleration.

File: Jupyter/profit_multistrat_plot.ipynb [hunk 2]
```diff
@@ -0,0 +1,4960 @@
...
+          49.917893,
+          24.872442285683512,
+          49.84308296330628
+         ]
+        },
+        {
+         "name": "BinHV45",
+         "type": "scatter",
+         "x": [
+          "2018-01-05 04:20:00",
```
Comment (vertti) [inline Jupyter/profit_multistrat_plot.ipynb]: This script seems to hardcode huge amount of data instead of loading it from tickers or our test data sets. It will be a nightmare to keep up to date.

**Comment by Alocaly** [comment]
Hey, 

Some clarifications :
1) I'm new to Github, so I only wanted the first commit to appear in this PR, that fixed the plot datatframe and the plot profit features. 
So I need to revert the other commits, learn how to really use github correctly, and  provide some clean PR.
I'm sincerely sorry for this...
2) Jupyter vs non Jupyter is completely a legitimate debate. I feel like I'm faster with experimenting, and drawing some graph and strat behavior / results in Jupyter than with plain python, and the fact that you can keep your results in one single place is great.
Several points  : 
-it's completely WIP for now, and was not intended to be in this PR, so yes, it's hardcoded and messy, and probably bugged, but that was my first step
-I'm using as much as possible code from freqtrade directly. So the data, for instance, are loaded from the  load_date function ( and it uses downloaded history if avalaible )
- if my results proved to be useful (that I hope ), I would like it to be an optional part of Freqtrade, like a separate folder, with additional install to add.
This way, it will be free for the servers / VM in the cloud.
But for people that want to experiment, they would have some material to help them.
This of course need to be discussed, I can still extract python from my Jupyter and include them in scripts...

Thanks, 

For now my first step is to understand how to work with github...

File: freqtrade/optimize/backtesting.py [hunk 3]
```diff
@@ -196,21 +196,22 @@ def backtest(self, args: Dict) -> DataFrame:
...
+                    trade_entry = trade_entry + ( next_date,)
                     trades.append(trade_entry)
                     if record:
                         # Note, need to be json.dump friendly
                         # record a tuple of pair, current_profit_percent,
                         # entry-date, duration
                         records.append((pair, trade_entry[1],
-                                        row.date.strftime('%s'),
-                                        row2.date.strftime('%s'),
+                                        str(int(row.date.timestamp())),
```
[the remark this answers is not in the document]
Comment (Alocaly) [reply freqtrade/optimize/backtesting.py]: Actually, row.date.timestamp() returned somethign like '13334354.0'.
That's why.
Please note that row.date.strftime('%s') is not standard, and does not work on my machine ( win10 )

File: scripts/plot_dataframe.py (hunk 1 above)
Comment (Alocaly) [reply scripts/plot_dataframe.py]: On my Pc ( Win10 ), opening this file will use by default Edge, and the buy / sell graph was not displayed at all. 
Hence this change.
This has probably happened before, because the other graphs are rendered without WebGL ( like the candle and the Bollinger bands ).
So I think the impact on perf for just the few buy/ sell items is negligeable ( and it adds more compatibility )
